SECP-service/src/main/resources/assets/app/css/user-profile.css

```diff
@@ -0,0 +1,94 @@
+@import url("https://fonts.googleapis.com/css?family=Raleway:400,700");
```

Comment:
For some reason, the user profile page is changing the font styles for the elements in the navbar. Please address this issue.

Reply:
Please address the comment above. The navbar really looks weird.

Reply:
![image](https://user-images.githubusercontent.com/8702831/34629684-1013c84c-f237-11e7-832a-ce9928e9a339.png)
I dont see the navbar issue. Can you be a little more specific?

Reply:
All good now :)